app/pages/project/workflow-selection.jsx

```diff
@@ -12,12 +12,17 @@ class WorkflowSelection extends React.Component {
     super();
     this.state = {
       error: null,
-      loadingSelectedWorkflow: true
+      loadingSelectedWorkflow: false
```

Comment:
Why was this changed to false? It seems with this change the component would load on initial render https://github.com/zooniverse/Panoptes-Front-End/blob/97f8bc24ba926fa24ab6756b82727cd0d8efbd16/app/pages/project/workflow-selection.jsx#L190

Reply:
If you set this to true, you should notice that the classifier never loads.

Reply:
Ahh. You are correct.

Reply:
If there’s already a valid workflow loaded, we set this to false and skip workflow loading completely.

Reply:
That could be set explicitly in `componentDidMount` if that makes the code easier to understand. I’ll leave it to you to change if you think that’s better.

Reply:
No, I think it's unnecessary to have an additional setState in `componentDidMount` if this is the property you want.